README.md

```diff
@@ -1,268 +1,143 @@
-    
-### pyfirmata ###
-    $ sudo pip install pyfirmata
-
-## Installation on Mac OS X ##
-
-On OS X, we use MacPorts to obtain the requisite libaries for at least part of the stack. 
-
-forseti2 is depdendent on several external libraries. Follow these instructions on OS X to install these dependencies. 
-
-### python ###
-
-OS X comes with its own distribution of python. However, you should install the MacPorts distribution to be able to easily install external libraries on top of python. 
-
-    $ sudo port install python27
-    $ sudo port install py27-pip
-    $ sudo port select --set python python27
-
-### Flask ###
-
-    $ sudo pip install flask
-
-### LCM ####
-
-Refer to the LCM install instructions at [lcm.googlecode.com wiki](https://code.google.com/p/lcm/wiki/BuildInstructions). 
-
-Once LCM is built, move `lcm-1.0.0/lcm-java/lcm.jar` into this directory to take advantage of `lcm-spy.sh` packet introspection.
-
-### pygame ###
-    $ sudo port install py27-game
-    
-### arduino and firmata ###
-
-There's no port avaialble for arduino or firmata. 
-
-## Nodes and their Functions ##
-
-### timer.py ###
-
+forseti2 (Named after the Norse god of justice) is PiE's third* iteration of a Field Control software. It has been in use from 2014-2017, with substantial changes along the way.
+
+## Deploying Driver Stations
+Driver Stations can be downloaded from PieCentral's [Releases section](https://github.com/pioneers/PieCentral/releases), or built locally using the `dawn/fieldcontrol` branch.
+
+### Installing for MacOS and GNU/Linux
+ 1. Download the `darwin-x64` or `linux-x64` release anywhere (preferably desktop)
+ 2. In Terminal, run this (but remember to update the station number and controller's address)
+ ```bash
+ echo "Type Bridge Address:";
+ read address;
+ if [ -z "$address" ]; then
+     address="192.168.128.138";
+ fi
+ echo -n $address > ~/Desktop/bridge_address.txt;
+ 
+ echo "Bridge Station (0-1 For Blue, 2-3 For Gold):"
+ read station;
+ if [ -z "$station" ]; then
+     station=0;
+ fi
+ echo -n $station > ~/Desktop/station_number.txt;
+ ```
+ 3. Run Field Control Dawn
+
+### Installing for Windows
+ 1. Download the `win32-ia32` release anywhere
+ 2. On the Desktop:
+   * Create the file `bridge_address.txt` and write the Controller's IP address
+   * Create the file `station_number.txt` and write the station number you want to be (0, 1, 2, 3 => blue1, blue2, gold1, gold2)
+
+## Deploying Field Controller
+forseti2 is dependent on external libraries, so there will be some OS-dependent instructions.
+
+### [Lightweight Communications and Marshalling (LCM)](https://lcm-proj.github.io/)
+1. Follow Online instructions on how to build it on your system.
+2. Once LCM is built, move `lcm.jar` into the forseti2 directory to take advantage of `lcm-spy.sh` packet introspection.
+
+### Installing for Ubuntu
+1. Install LCM as detailed above
+2. Install other packages:
+```bash
+sudo apt-get install python-pygame;
```

Comment:
I don't think forseti needs pygame anymore. It used to be used for xbox controllers (as part of live scoring), and maybe for UI -- but now we've standardized on doing UI in the browser